frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/queries.ts

```diff
@@ -0,0 +1,10 @@
+export const enum CapacityQueryLabels {
```

Comment:
should be moved here https://github.com/openshift/console/blob/master/frontend/packages/ceph-storage-plugin/src/constants/queries.ts

Reply:
Acknowledged...